subprojects/docs/src/docs/userguide/potentialTraps.adoc

```diff
@@ -25,11 +25,11 @@ For Gradle users it is important to understand how Groovy deals with script vari
 ++++
 <sample xmlns:xi="http://www.w3.org/2001/XInclude" id="scope" dir="userguide/tutorial" title="Variables scope: local and script wide">
             <sourcefile file="scope.groovy"/>
-            <output args=""/>
+            <output args="scope.groovy" executable="groovy" />
         </sample>
 ++++
 
-Variables which are declared with a type modifier are visible within closures but not visible within methods. This is a heavily discussed behavior in the Groovy community.footnote:[One of those discussions can be found here: http://groovy.329449.n5.nabble.com/script-scoping-question-td355887.html[] ] 
+Variables which are declared with a type modifier are visible within closures but not visible within methods. This is a heavily discussed behavior in the Groovy community.footnote:[One of those discussions can be found here: http://groovy.329449.n5.nabble.com/script-scoping-question-td355887.html[] ]
```

Comment:
This is a dead link.  I think it would be fine to shorten this and remove the sentence about this being a heavily discussed thing:
> Variables which are declared with a type modifier are visible within closures but not visible within methods.

Reply:
done